netket/operator/_local_operator.py

```diff
@@ -120,7 +128,23 @@ def resize(
     return new_arr
 
 
-def _reorder_matrix(hi, mat, acting_on):
+def _reorder_kronecker_product(hi, mat, acting_on):
```

Comment:
is this in preparation of having unsorted hilberts? or is it already working?

Reply:
oh I see it's the old thing...

Reply:
no, just a better name and a docstring